doc/latex.rst

```diff
@@ -65,6 +65,43 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
    }
    latex_show_urls = 'footnote'
 
+.. _latex-documentclass:
+
+The LaTeX document class
+------------------------
+
+LaTeX document classes are files (with extension ``.cls``) containing LaTeX
+macros, often devoted to custom layout definitions.  They are loaded via
+the ``\documentclass{<classname>}`` line in a LaTeX source file.
+
+Sphinx provides two LaTeX classes ``sphinxhowto`` and ``sphinxmanual`` of its
+own (the two ``.cls`` files are automatically copied over to the LaTeX build
+repertory).  Which one is used depends on whether ``'howto'`` or ``'manual'``
+is encountered in the :confval:`latex_documents` variable (the default uses
+``'manual'``).  Any string distinct from these two is interpreted as the name
+of a LaTeX class which will be loaded in place of the Sphinx custom ones.  In
+such case the ``\sphinxmaketitle``, ``\sphinxtableofcontents``,
+``sphinxthebibliography``, and ``sphinxtheindex`` fall back to the meaning of
+the LaTeX macros and environments without ``sphinx`` prefix.
+
+The Sphinx LaTeX classes are slim wrappers of standard classes: respectively
+the ``article`` and ``report`` ones (Japanese documents use ``jreport`` and
+``jsbook``).  It is possible to modify these base classes via the
```

Comment:
It is possible to alter the mapping between Sphinx and base classes via the :confval:`latex_docclass` dictionary.